docs/standard/base-types/regular-expression-language-quick-reference.md

```diff
@@ -53,18 +53,18 @@ A character class matches any one of a set of characters. Character classes incl
-|`\W`|Matches any non-word character.|`\W`|`" "`, `"."` in `"ID A1.3"`|
-|`\s`|Matches any white-space character.|`\w\s`|`"D "` in `"ID A1.3"`|
-|`\S`|Matches any non-white-space character.|`\s\S`|`" _"` in `"int __ctr"`|
-|`\d`|Matches any decimal digit.|`\d`|`"4"` in `"4 = IV"`|
-|`\D`|Matches any character other than a decimal digit.|`\D`|`" "`, `"="`, `" "`, `"I"`, `"V"` in `"4 = IV"`|
+|`[` *character_group* `]`|Matches any [single character in *character_group*](character-classes-in-regular-expressions.md#positive-character-group--). By default, the match is case-sensitive.|`[ae]`|`"a"` in `"gray"`<br /><br /> `"a"`, `"e"` in `"lane"`|
+|`[^` *character_group* `]`|Negation: Matches any [single character that is not in *character_group*](character-classes-in-regular-expressions.md#negative-character-group-). By default, characters in *character_group* are case-sensitive.|`[^aei]`|`"r"`, `"g"`, `"n"` in `"reign"`|
+|`[` *first* `-` *last* `]`|Character range: Matches any [single character in the range from *first* to *last*](character-classes-in-regular-expressions.md#positive-character-group--).|`[A-Z]`|`"A"`, `"B"` in `"AB123"`|
+|`.`|Wildcard: Matches any [single character except `\n`](character-classes-in-regular-expressions.md#any-character-).<br /><br /> To match a literal period character (. or `\u002E`), you must precede it with the escape character (`\.`).|`a.e`|`"ave"` in `"nave"`<br /><br /> `"ate"` in `"water"`|
+|`\p{` *name* `}`|Matches [any single character in the Unicode general category or named block specified by *name*](character-classes-in-regular-expressions.md#unicode-category-or-unicode-block-p).|`\p{Lu}`<br /><br /> `\p{IsCyrillic}`|`"C"`, `"L"` in `"City Lights"`<br /><br /> `"Д"`, `"Ж"` in `"ДЖem"`|
+|`\P{` *name* `}`|Matches [any single character that is not in the Unicode general category or named block specified by *name*](character-classes-in-regular-expressions.md#negative-unicode-category-or-unicode-block-p).|`\P{Lu}`<br /><br /> `\P{IsCyrillic}`|`"i"`, `"t"`, `"y"` in `"City"`<br /><br /> `"e"`, `"m"` in `"ДЖem"`|
+|`\w`|Matches any [word character](character-classes-in-regular-expressions.md#word-character-w)|`\w`|`"I"`, `"D"`, `"A"`, `"1"`, `"3"` in `"ID A1.3"`|
+|`\W`|Matches any [non-word character](character-classes-in-regular-expressions.md#non-word-character-w).|`\W`|`" "`, `"."` in `"ID A1.3"`|
+|`\s`|Matches any [white-space character](character-classes-in-regular-expressions.md#whitespace-character-s).|`\w\s`|`"D "` in `"ID A1.3"`|
+|`\S`|Matches any [non-white-space character](character-classes-in-regular-expressions.md#non-whitespace-character-s).|`\s\S`|`" _"` in `"int __ctr"`|
+|`\d`|Matches any [decimal digit](character-classes-in-regular-expressions.md#decimal-digit-character-d).|`\d`|`"4"` in `"4 = IV"`|
+|`\D`|Matches any [character other than a decimal digit](character-classes-in-regular-expressions.md#non-digit-character-d).|`\D`|`" "`, `"="`, `" "`, `"I"`, `"V"` in `"4 = IV"`|
```

Comment:
@gfoidl I redid these to do deep links to the more info article.

Reply:
This is a great idea 👍🏻